pkg/standalone-utils/contracts/test/WeightedLPOracleMock.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { AggregatorV3Interface } from "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol";
+
+import { IWeightedPool } from "@balancer-labs/v3-interfaces/contracts/pool-weighted/IWeightedPool.sol";
+import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
+
+import { WeightedLPOracle } from "../WeightedLPOracle.sol";
+
+contract WeightedLPOracleMock is WeightedLPOracle {
```

Comment:
Might be an overkill, but maybe some of these getters are actually useful in the production contract. Let's double check with integrations eventually; fine to leave it like this for now.